internal/tailer/logstream/logstream.go

```diff
@@ -48,6 +49,13 @@ const defaultReadBufferSize = 4096
 // channel.  `seekToStart` is only used for testing and only works for regular
 // files that can be seeked.
 func New(ctx context.Context, wg *sync.WaitGroup, waker waker.Waker, pathname string, lines chan<- *logline.LogLine, streamFromStart bool) (LogStream, error) {
+	protocol, spec := ParsePathName(pathname)
```

Comment:
I originally started with url.Parse, however, once I read the previously linked issue, with the `unixgram:` prefix, it wasn't specified as `unixgram://`, and not super keen on `unixgram:///var/run/socket`.

Reply:
Do you mean that because I didn't say explicitly `unixgram://` in the issue?  I think that's not a great reason to not use it.

I won't object to leaving the code as is for now though, but I will likely change it to use url.Parse once submitted.

Reply:
Yeah, exactly - then the crunchiness of unixgram:///file/path.  It's fine though, I'd change it if that's your preference.

Reply:
Yeah I think reusing a common naming pattern is a good move, I don't want to invent another scheme.  (`mtail` is already its own language :)  I think that other tools use unixgram:// already, too, I want to believe that rsyslog uses that naming scheme, but I haven't gone to check.

I am also hoping that url.parse can read a scheme-less URL and default to being a file:// location.